lib/Dialect/SV/Transforms/SVExtractTestCode.cpp

```diff
@@ -204,7 +204,11 @@ static hw::HWModuleOp createModuleForCut(hw::HWModuleOp op,
   {
     auto srcPorts = op.getInputNames();
     for (auto port : llvm::enumerate(realInputs)) {
-      auto name = getNameForPort(port.value(), srcPorts);
+      // Append the port index to create unique names.
+      auto name =
+          b.getStringAttr(getNameForPort(port.value(), srcPorts).getValue() +
```

Comment:
Maybe  can we do something simpler and more stable naming like this?
```suggestion
          auto name = getNameForPort(port.value(), srcPorts);
          name = name.getValue().empty()? b.getStringAttr("arg" + Twine(port.index()));
```

Reply:
Also “arg” seems to be a bit weird to me, can we do name this “port”?